shimmingtoolbox/unwrap/prelude.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Wrapper to FSL Prelude (https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FUGUE/Guide#PRELUDE_.28phase_unwrapping.29)
+"""
+
+import numpy as np
+import os
+import nibabel as nib
+import subprocess
+
+
+def prelude(wrapped_phase, mag, affine, mask=np.array([-1]), path_2_unwrapped_phase='./unwrapped_phase.nii',
+            is_unwrapping_in_2d=True, is_saving_nii=False):
```

Comment:
i would remove `is_saving_nii ` and check if logger is in DEBUG mode. If it is, save those outputs.

Reply:
I'll remove the feature entirely, if the niis are required, they can directly be saved in the parent function. Using DEBUG requires resaving the niis since they are in a temporary directory.

Reply:
> I'll remove the feature entirely, if the niis are required, they can directly be saved in the parent function. Using DEBUG requires resaving the niis since they are in a temporary directory.

and in fact, the niftis are _already_ saved, so no need to re-save them. The temp folders usually stay there for several hours (or weeks, depending on the OS config), which is enough for debugging